### PR TITLE
Dev45

### PR DESCRIPTION
**Pull request by Dafinia**
eh it's not like I can do better anyway

### CODE REVIEW

File: JavaHomeworks/1.4 QuadraticEquationRoots/QuadraticEquationRoots.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ */
+public class QuadraticEquationRoots {
+  /**
+   * A main method of QuadraticEquationRoots class
+   *
+   * @param args command line args as coeffs of a quadratic equation
+   */
+  public static void main(String[] args) {
+    // since it's pure math I think it should be ok to use a, b, c as variable names
+    double a;
```
Comment (AlexanderSoroka) [inline JavaHomeworks/1.4 QuadraticEquationRoots/QuadraticEquationRoots.java]: It must be declared closer to it first usage

File: JavaHomeworks/1.4 QuadraticEquationRoots/QuadraticEquationRoots.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ * Quadratic equation solver. Takes 3 arguments from command line and treats them as coefficients a,
+ * b and c of the equation ax^2 + bx +c = 0 and then solves for x.
+ */
+public class QuadraticEquationRoots {
+  /**
+   * A main method of QuadraticEquationRoots class
+   *
+   * @param args command line args as coeffs of a quadratic equation
+   */
+  public static void main(String[] args) {
```
Comment (AlexanderSoroka) [inline JavaHomeworks/1.4 QuadraticEquationRoots/QuadraticEquationRoots.java]: It's big and hard to read - split it to smaller methods

File: JavaHomeworks/1.4 QuadraticEquationRoots/QuadraticEquationRoots.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    double c;
+
+    // checking if number of arguments is ok
+    if (args.length != 3) {
+      System.out.println("Invalid number of arguments.");
+      System.exit(1);
+    }
+    // checking if parsable to double
+    try {
+      a = Double.parseDouble(args[0]);
```
Comment (AlexanderSoroka) [inline JavaHomeworks/1.4 QuadraticEquationRoots/QuadraticEquationRoots.java]: What if user inputs 2e2048 as argument value?

File: JavaHomeworks/1.4 QuadraticEquationRoots/QuadraticEquationRoots.java (hunk 2 above)
Comment (Dafinia) [inline JavaHomeworks/1.4 QuadraticEquationRoots/QuadraticEquationRoots.java]: That would imply that I know how to use exceptions. I don't.

Comment (AlexanderSoroka) [inline JavaHomeworks/1.4 QuadraticEquationRoots/QuadraticEquationRoots.java]: Doesn't matter - you could split it without deep knowledge what is exception and how it works

**Pull request closed without merging**